### PR TITLE
Doc addition: How to compile libs with different debug level

### PR DESCRIPTION
**Pull request by PilnyTomas**
## Description of Change

- Added new file `docs/source/guides/core_debug.rst` explaining in detail how to modify libuilder to compile with different log level.
- Added minimal tip + link into FAQ

## Tests scenarios
Tested only an rst preview, but I don't know how will it actually draw on a web page.

## Related links
https://github.com/espressif/esp32-arduino-lib-builder/issues/79

### CODE REVIEW

**Review by pedrominatel — APPROVED** [review]

**Review by pedrominatel — APPROVED** [review]
LGTM

File: docs/source/guides/core_debug.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+   CONFIG_LOG_DEFAULT_LEVEL_NONE=y # No output
+   CONFIG_LOG_DEFAULT_LEVEL_ERROR=y # Errors - default
+   CONFIG_LOG_DEFAULT_LEVEL_WARN=y # Warnings
+   CONFIG_LOG_DEFAULT_LEVEL_INFO=y # Info
+   CONFIG_LOG_DEFAULT_LEVEL_DEBUG=y # Debug
+   CONFIG_LOG_DEFAULT_LEVEL_VERBOSE=y # Verbose
+
+Then simply build the libs for all SoCs or one specific SoC. Note that building for all SoCs takes a lot of time, so if you are working only with specific SoC(s), build only for those.
+
+
```
Comment (VojtechBartoska) [inline docs/source/guides/core_debug.rst:25]: Here are two empty lines probably by typo

**Review by VojtechBartoska — APPROVED** [review]
LGTM

File: docs/source/guides/core_debug.rst (hunk 1 above)
Comment (me-no-dev) [reply docs/source/guides/core_debug.rst:25]: @PilnyTomas could you fix this please?

Comment (PilnyTomas) [reply docs/source/guides/core_debug.rst]: done in [cb3ac63](https://github.com/espressif/arduino-esp32/pull/7193/commits/cb3ac630c294eabddf9adea84ba5a4ecfe2c02c7)

**Pull request merged**